Initial rewrite of windows install (#40999)
dotnet/docs: docs/core/install/windows.md

### Users and Developers

- [.NET Installer](#net-installer)

  Install .NET with a Windows Installer package, which is an executable that you run. This method can install the SDK and Runtime. Installs are performed system-wide.

- [Windows Package Manager (WinGet)](#install-with-windows-package-manager-winget)

  Use **WinGet** to install .NET when you want to manage .NET through the command line. This method can install the SDK and Runtime. Installs are performed system-wide.

- [PowerShell](#install-with-powershell)

  A PowerShell script that can automate the install of the SDK or Runtime. You can choose which version of .NET to install.

## Supported versions

The following table is a list of currently supported .NET releases and the versions of Windows they're supported on. These versions remain supported until either the version of [.NET reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Windows reaches end-of-life](https://support.microsoft.com/help/13853/windows-lifecycle-fact-sheet).

> [!TIP]

> > As a reminder, this table applies to modern .NET (as opposed to .NET Framework). .NET Framework downloads can be found [here](https://dotnet.microsoft.com/download/dotnet-framework).

Author: ```suggestion
> As a reminder, this table applies to modern .NET (as opposed to .NET Framework). To install .NET Framework, see the [.NET Framework Installation guide](../../framework/install/index.md).
```

| **ASP.NET Core Runtime** | No                            | No                            | Yes                           |
| **.NET SDK**             | Yes                           | Yes                           | Yes                           |

> 

Reviewer: Blank line failing the lint

01. Select **Help** > **About Microsoft Visual Studio**.
01. Read the version number from the **About** dialog.

For more information about, see [.NET SDK, MSBuild, and Visual Studio versioning](../porting/versioning-sdk-msbuild-vs.md).

## Install with Visual Studio Code

> Visual Studio Code uses the versions of .NET already installed on your system. Remember, Visual Studio installs its own copy of .NET, so you must install .NET using either [.NET Installer](#net-installer) or [Windows Package Manager](#install-with-windows-package-manager-winget).

Reviewer: ```suggestion
Visual Studio Code uses the versions of .NET already installed on your system. Install .NET using either [.NET Installer](#net-installer) or [Windows Package Manager](#install-with-windows-package-manager-winget). Visual Studio installs its own copy of .NET that can't be used by Visual Studio Code.
```

## Install with Windows Package Manager (WinGet)

You can install and manage .NET through the Windows Package Manager service, using the `winget.exe` tool. For more information about how to install and use **WinGet**, see [Use the winget tool to install and manage applications](/windows/package-manager/winget/).

If you're installing .NET system-wide, install with administrative privileges.

The .NET WinGet packages are:

- `Microsoft.DotNet.Runtime.8`&mdash;.NET Runtime 8.0.
- `Microsoft.DotNet.AspNetCore.8`&mdash;ASP.NET Core Runtime 8.0
- `Microsoft.DotNet.DesktopRuntime.8`&mdash;.NET Desktop Runtime 8.0
- `Microsoft.DotNet.SDK.8`&mdash;.NET SDK 8.0

### Install the SDK

If you install the SDK, you don't need to install the corresponding runtime.

01. [Install WinGet](/windows/package-manager/winget/#install-winget).
01. Open a terminal, such as PowerShell or `cmd.exe`.
01. Run the `winget install` command and pass the name of the SDK package:

    ```cmd
    winget install Microsoft.DotNet.SDK.8
    ```

### Install the runtime

> There are different runtimes you can install. Refer to the [Which runtime should you install](#which-runtime-should-you-install) section to understand what's included with each runtime.

Reviewer: ```suggestion
There are different runtimes you can install. Refer to the [Choose the correct runtime](#choose-the-correct-runtime) section to understand what's included with each runtime.
```
